deps/build.jl

```diff
@@ -14,17 +16,47 @@ function validate_libFTD2XX_version(name, handle)
     v = Ref{Cuint}()
     s = ccall(f, Culong, (Ref{Cuint},), v)
     s == C_NULL && return false
-    return v[] >= 0x00021228
+    if Compat.Sys.iswindows()
+        return v[] >= 0x00021228
```

Comment:
What does this do?

Reply:
Only windows has a version number function, so we check it, but we don't on OS X